src/pages/home/report/ReportActionItemDate.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {memo} from 'react';
+import {OnyxEntry} from 'react-native-onyx';
+import {withCurrentDate} from '@components/OnyxProvider';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@styles/useThemeStyles';
+
+type ReportActionItemDateOnyxProps = {
+    // eslint-disable-next-line react/no-unused-prop-types
+    currentDate: OnyxEntry<string>;
```

Comment:
What is the purpose of this prop?

Reply:
I agree, @JKobrynski let's move the comments from JS file to this prop.

```ts
type ReportActionItemDateOnyxProps = {
    /** 
     * UTC timestamp for when the action was created.
     * This Onyx prop is hooked to the current date so that relative times can update when necessary
     * e.g. past midnight.
    */
    // eslint-disable-next-line react/no-unused-prop-types
    currentDate: OnyxEntry<string>;
};
```

Reply:
Done!